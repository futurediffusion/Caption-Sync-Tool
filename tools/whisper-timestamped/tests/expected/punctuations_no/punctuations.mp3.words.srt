1
00:00:00,380 --> 00:00:01,100
Dis-moi

2
00:00:01,280 --> 00:00:01,500
est-ce

3
00:00:01,500 --> 00:00:01,660
que

4
00:00:01,660 --> 00:00:02,040
l'avion

5
00:00:02,040 --> 00:00:02,760
vole

