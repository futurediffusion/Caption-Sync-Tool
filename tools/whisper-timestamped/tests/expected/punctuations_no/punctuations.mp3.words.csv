Dis-moi,0.38,1.1
est-ce,1.28,1.5
que,1.5,1.66
l'avion,1.66,2.04
vole,2.04,2.76
